### PR TITLE
Added new option for cert generate command to overwrite existing certs

### PR DESCRIPTION
**Pull request by prabhu-das**
Added changes to support- cert generate command overwrites existing certs if destination files already exist.
Introduced new option `--overwrite-certs`

### CODE REVIEW

File: lib/chef/knife/windows_cert_generate.rb [hunk 1]
```diff
@@ -56,6 +56,12 @@ class WindowsCertGenerate < Knife
         :long => "--cert-passphrase PASSWORD",
         :description => "Password for certificate."
 
+      option :overwrite_certs,
```
Comment (adamedx) [inline lib/chef/knife/windows_cert_generate.rb]: I actually don't think we need this flag -- it's something we could add back later if users ask for it.

**Comment by btm** [comment]
Agree with @adamedx. If anything we'd probably roll this functionality into something like a "--force" flag.

Although, we should probably raise an error and exit if the files exist. It looks like we're blindly overwriting them right now.

**Comment by adamedx** [comment]
Yeah, I think there's a line that does do what you suggest @btm -- overwrite_existing_certs! But we could probably just change the name of that method and also have it not read from config since we are getting rid of the config option.

**Comment by prabhu-das** [comment]
@btm @adamedx 
Fixed review comments.
Thank you.

File: lib/chef/knife/windows_cert_generate.rb [hunk 2]
```diff
@@ -108,11 +108,32 @@ def write_certificate_to_file cert, file_path, rsa_key
         File.open(file_path + ".b64", "wb") { |f| f.print Base64.strict_encode64(pfx.to_der) }
       end
 
+      def is_other_certificates_present?(file_path)
+        is_certs_exists = false
```
Comment (NimishaS) [inline lib/chef/knife/windows_cert_generate.rb]: Suggestion for method and variable names:
1. `is_other_certificates_present?` method name can be `certificates_already_exist?`
2. `is_certs_exists` variable name can be `certs_exist`

Comment (adamedx) [inline lib/chef/knife/windows_cert_generate.rb]: Agree with @NimishaS's feedback.

Comment (prabhu-das) [inline lib/chef/knife/windows_cert_generate.rb]: Fixed comments.
Thanks!

**Pull request merged**